tests/app/app_routes/cancel_restart/test_cancel_restart_routes.py

```
@@ -85,7 +85,10 @@ def update_status(self, task_id: str, status: str) -> None:
         "src.main_app.app_routes.cancel_restart.routes.current_user", lambda: types.SimpleNamespace(username="user")
     )
     monkeypatch.setattr("src.main_app.app_routes.cancel_restart.routes.active_coordinators", lambda: ["user"])
-    monkeypatch.setattr("src.main_app.app_routes.cancel_restart.routes.get_cancel_event", lambda task_id: DummyEvent())
+    monkeypatch.setattr(
+        "src.main_app.app_routes.cancel_restart.routes.get_cancel_event",
+        lambda task_id, store=None: DummyEvent()
```

Ruff (0.15.1)

[warning] Unused lambda argument: `task_id`

(ARG005)